examples/cluster-test/cluster-test.cpp

```diff
@@ -0,0 +1,113 @@
+/**
```

Comment:
This is just a quick and dirty implementation to visualize Verlet cluster. A VerletClusterLists container is filled and the generated clusters are printed into a vtu files. If unwanted it can be deleted.